packages/components/src/ui/ariakit-tooltip/test/index.tsx

```diff
@@ -0,0 +1,330 @@
+		).toHaveFocus();
+
+		await waitFor( () =>
+			expect(
+				screen.getByRole( 'tooltip', { name: /tooltip text/i } )
+			).toBeVisible()
+		);
+
+		await user.tab();
+
+		expect(
+			screen.queryByRole( 'tooltip', { name: /tooltip text/i } )
+		).not.toBeInTheDocument();
+
+		// Wait for the tooltip element to be positioned (aligned with the button)
+		// await waitFor( () =>
+		// 	expect(
+		// 		getWrappingPopoverElement( screen.getByText( 'tooltip text' ) )
+		// 	).toBePositionedPopover()
+		// );
+	} );
+
+	it( 'should render the tooltip when the tooltip anchor is hovered', async () => {
+		const user = userEvent.setup();
+
+		render( <ToolTip { ...props } /> );
+
+		const button = screen.getByRole( 'button', { name: /Button/i } );
+
+		await user.hover( button );
+
+		await waitFor( () =>
+			expect(
+				screen.getByRole( 'tooltip', { name: /tooltip text/i } )
+			).toBeVisible()
+		);
+
+		// Wait for the tooltip element to be positioned (aligned with the button)
+		// await waitFor( () =>
+		// 	expect(
+		// 		getWrappingPopoverElement( screen.getByText( 'tooltip text' ) )
+		// 	).toBePositionedPopover()
+		// );
+
+		await user.unhover( button );
+
+		expect(
+			screen.queryByRole( 'tooltip', { name: /tooltip text/i } )
+		).not.toBeInTheDocument();
+	} );
+
+	it( 'should not show tooltip on focus as result of mouse click', async () => {
+		const user = userEvent.setup();
+
+		render( <ToolTip { ...props } /> );
+
+		await user.click( screen.getByRole( 'button', { name: /Button/i } ) );
+
+		expect(
+			screen.queryByRole( 'tooltip', { name: /tooltip text/i } )
+		).not.toBeInTheDocument();
+	} );
+
+	it( 'should respect custom delay prop when showing tooltip', async () => {
+		const user = userEvent.setup( { delay: TOOLTIP_DELAY } );
+		const CUSTOM_DELAY = TOOLTIP_DELAY + 25;
+
+		render( <ToolTip { ...props } delay={ CUSTOM_DELAY } /> );
+
+		const button = screen.getByRole( 'button', { name: /Button/i } );
+
+		await user.hover( button );
+
+		expect(
+			screen.queryByRole( 'tooltip', { name: /tooltip text/i } )
+		).not.toBeInTheDocument();
+
+		await waitFor( () =>
+			expect(
+				screen.getByRole( 'tooltip', { name: /tooltip text/i } )
+			).toBeVisible()
+		);
+
+		await user.unhover( button );
+
+		expect(
+			screen.queryByRole( 'tooltip', { name: /tooltip text/i } )
+		).not.toBeInTheDocument();
+
+		// Wait for the tooltip element to be positioned (aligned with the button)
+		// await waitFor( () =>
+		// 	expect(
+		// 		getWrappingPopoverElement( screen.getByText( 'tooltip text' ) )
+		// 	).toBePositionedPopover()
+		// );
+	} );
+
+	it( 'should show tooltip when an element is aria-disabled', async () => {
+		const user = userEvent.setup();
+
+		render(
+			<ToolTip { ...props }>
+				<Button aria-disabled>Button</Button>
+			</ToolTip>
+		);
+
+		const button = screen.getByRole( 'button', { name: /Button/i } );
+
+		expect( button ).toBeVisible();
+		//expect( button ).toBeDisabled();
```

Comment:
In the legacy tooltip, the tooltip will show if an element is disabled. From what I've seen, it shouldn't show unless `aria-disabled` is added instead. In order to be backwards compatible, we may need to add a temporary solution to be deprecated 🤔